### DISCUSSION TITLE
har是否适用于gpt3.5

### DISCUSSION THREAD

**Discussion by xbl916** [linweiyuan/go-chatgpt-api#275] [Q&A] [answered]
我使用账号登录获取access token，提示Error getting auth Arkose token。更新最新镜像后，按https://linweiyuan.github.io/2023/06/24/%E5%A6%82%E4%BD%95%E7%94%9F%E6%88%90-GPT-4-arkose-token.html 中的视频教程获取了chat.openai.com.har并进行挂载，尝试登录仍然提示
![image](https://github.com/linweiyuan/go-chatgpt-api/assets/24452427/1b3e0534-2480-41f9-a01f-5aef416955ee)
我没有plus，只使用gpt3.5

**Comment by linweiyuan** [comment]
官网退出登录，然后开始抓，从登录开始抓到进去系统

**Reply by xbl916** [reply]
不好意思 前面打错了 ，是没有plus.....我从登录开始抓仍然不行，还是一样的错误

**Comment by linweiyuan** [comment, marked as answer]
要选上保留历史，不然登录跳转就没了

**Reply by xbl916** [reply]
ok 搞定了 感谢

**Comment by shiyegao** [comment]
我这边点击render部署了一个服务，但是会使用时返回```{"errorMessage":"a valid HAR file required"}%```

render反应
![image](https://github.com/linweiyuan/go-chatgpt-api/assets/45991529/ab4c3a21-87aa-4e7f-8b78-47664983301f)

使用指令：
```
curl -XPOST -H "Content-type: application/json" -d '{
  "username": "xxx@mail.com",
  "password": "xxxxx"
}' 'https://go-chatgpt-api-xxxx.onrender.com/chatgpt/login'
```

请问我应该怎么改才能正常使用呢

**Reply by linweiyuan** [reply]
render 不能用登录接口

**Reply by shiyegao** [reply]
ok，感谢。

我这边也试了一下go build main.go编译出来的可执行文件，但是也会遇到同样的错误，也是一样的原因么？

向请教一下怎么才能登录接口呢

**Reply by linweiyuan** [reply]
模仿博客视频抓登录的 har 包

**Reply by shiyegao** [reply]
我看了一下，好像是每个账号需要单独操作一下？请问有没有什么办法可以像之前一样，可以复用性地自动获取token呢？

（还是我理解错了，抓包可以流水线作业？）

**Reply by linweiyuan** [reply]
因为官网改了

方法有没有不知道，因为我已经不研究这个了，弃坑了

**Reply by shiyegao** [reply]
了解了，感谢

**Reply by shiyegao** [reply]
我这边找到了两个方法，和大家分享下

1. 比较方便的，但是接口有限流。
```
# pip install pandora-chatgpt

import pandora
accessToken = pandora.openai.auth(username, password).auth(False)
```

2. 比较麻烦的，物理外挂。

    用```pyautogui```,  ```pyperclip```等库模拟鼠标键盘的行为登录网站，亲测大概15s一个账号，且不会遇到429
